services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusResponse.kt

```diff
@@ -27,7 +27,7 @@ sealed interface HakemusDataResponse {
     val postalAddress: PostalAddress?
     val startTime: ZonedDateTime?
     val endTime: ZonedDateTime?
-    val areas: List<ApplicationArea>?
+    val areas: List<ApplicationArea>
```

Comment:
The UI seems to expect this array to be present, so I changed it to be non-nullable.